modules/virt-vmware-comparison.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/about_virt/about-virt.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-vmware-comparison_{context}"]
+= Comparing {VirtProductName} to {vmw-full}
+
+If you are familiar with {vmw-first}, the following table lists {VirtProductName} components that you can use to accomplish similar tasks. However, because {VirtProductName} is conceptually different from {vmw-short}, and much of its functionality comes from the underlying {product-title}, {VirtProductName} does not have direct alternatives for all {vmw-short} concepts or components.
+
+.Mapping of {vmw-short} concepts to their closest {VirtProductName} counterparts
+[options="header"]
+[cols="2,2,3"]
+|===
+|{vmw-short} concept |{VirtProductName} |Explanation
+
+|Datastore
+|Persistent volume (PV){nbsp}+ +
+Persistent volume claim (PVC)
+|Stores VM disks. A PV represents existing storage and is attached to a VM through a PVC. When created with the `ReadWriteMany` (RWX) access mode, PVCs can be mounted by multiple VMs simultaneously.
+
+|Dynamic Resource Scheduling (DRS)
+|Pod eviction policy{nbsp}+ +
+Descheduler
```

Comment:
Quick question about the table formatting:
are these extra `+` signs intentional?
![image](https://github.com/user-attachments/assets/6e9e88f3-34f7-46e6-a5ec-34bff772e893)

Because I think that just having the linebreaks like you do in the left column look better to me:
![image](https://github.com/user-attachments/assets/fef33538-ff4d-42be-bd88-16e0db7de85b)

Reply:
Hi @michaelryanpeter , the plus signs are intentional as both technologies in combination are a recommended alternative for either of the technologies listed in the first column. They are meant to be displayed and are followed by a line break to keep them cleanly on separate lines rather than breaking randomly.

Reply:
Heard. Thanks for clarifying. It wasn't clear to me when I read it. I can go ahead and merge this if you are still ready?